localizationkit/tests/placeholder_token_explanation.py

```diff
@@ -0,0 +1,95 @@
+"""Placeholder token explanation."""
+
+from typing import Any, Dict, List, Set
+
+from localizationkit.tests.test_case import LocalizationTestCase
+
+
+class PlaceholderTokenExplanation(LocalizationTestCase):
```

Comment:
I think this isn't a common requirement. We should probably make it opt-in.

Reply:
@PoonamBaxi This issue is still there. 

Note: I've had to remove everyone from the Outlook Mobile team for security purposes, so let me know if you can't push any more.

Reply:
Sorry not sure how I missed adding the class method for `is_opt_in(cls)` I fixed it locally but can't push
`Pushing to https://github.com/microsoft/localizationkit.git
remote: Permission to microsoft/localizationkit.git denied to PoonamBaxi.
fatal: unable to access 'https://github.com/microsoft/localizationkit.git/': The requested URL returned error: 403
`

Can you take a look please? Thanks!

Reply:
Fixed this now. For future contributions, you'll need to fork and open a PR that way (just as standard).

Reply:
@dalemyers Thanks for pushing the fix, I am not sure I followed your comment above. I had forked the master branch and used that for the change, also I fixed the issue locally but now cannot push to this branch anymore as my permissions are not working.